app/models/comment.rb

```diff
@@ -208,7 +208,7 @@ def readable_publish_date
   end
 
   def hover_publish_date
-    created_at.strftime("%B %e %Y %I:%M %P")
+    created_at.in_time_zone("Eastern Time (US & Canada)").strftime("%B %e %Y %I:%M %P %Z")
```

Comment:
What about `created_at.strftime("%B %e %Y %I:%M %P %Z")` ? Without converting the time to a US timezone?

Datetimes are always a tricky subject :D

Reply:
It's giving the UTC format (August 24 2018 12:34 pm UTC). I thought better display it in a friendly format for the user to converted to Eastern Time. But I can remove the timezone from that code if UTC is okay.

And yeah they are very tricky have to be cautious about what we display. 😖

Reply:
What I meant initially was to add explicitly `UTC`, with `%Z` as you did. Because if you don't tell the user what timezone the datetime is they will assume is their local one or, most likely, will end up asking "wait, in which timezone is this date? because it doesn't match my clock".

I wouldn't convert to an arbitrary timezone unless that's what @benhalpern and the others think it's best.

I might be the only one but if you tell me it's 12:34 UTC I know exactly what time it refers to my local time here in Europe (apart from actually looking at the clock :D), if you tell me is 12:34 Eastern Time Canada I need to ask Google, because I never remember 🗡 

You see, it's a simple change but a complicated issue.

A more comprehensive solution would be to have a user setting where someone can just decide what timezone they want and anonymous users get UTC or something, but that's quite a bigger change and it probably requires handling timezones both on the client and the server side.

Reply:
@rhymes Thanks for the insight 😃. I have modified it to UTC now. 

I too thought of a solution that gives the option for user to select their timezone and we storing it in our database. Then we accordingly convert time for the user so that he views the comment in his/her timezone itself. This would be an option that doesn't involve javascript and things can be done on the server side itself.